fuzz/tls_init/fuzz_init.cc

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2021 Intel Corporation
+ * Copyright (c) 2020-2021 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+extern "C"{
+#include <stdint.h>
+#include <stddef.h>
+#include <string.h>
+#include "rats-tls/api.h"
+#include "rats-tls/log.h"
+#include "rats-tls/claim.h"
+#include "internal/core.h"
+}
+#include <fuzzer/FuzzedDataProvider.h>
+#include <vector>
```

Comment:
Does the `libfuzzer-dev` package need to be installed? Please explain in the readme.

Reply:
Do we need to add a fuzz compilation option? When the flag is on, the fuzz folder is compiled.

Reply:
install the new version of clang, libfuzzer is included in it.

Reply:
> Do we need to add a fuzz compilation option? When the flag is on, the fuzz folder is compiled.

if needed, I would change the top CMakeLists, and add a -DFUZZ for this. now I just use add_subdictionary() to build fuzz in any condition.